app_ldap_test.go

```diff
@@ -67,7 +72,7 @@ func createYtsaurusGroup(name string) YtsaurusGroup {
 		ytName = strings.Replace(ytName, replacement.From, replacement.To, -1)
 	}
 	return YtsaurusGroup{Name: name, SourceRaw: map[string]any{
-		"groupname": originalName,
+		"groupname": ytName,
```

Comment:
It looks like YtsaurusGroup.Name should be equal to ytName, and SourceRow["groupname"] should be equal to originalName